memory_test.go

```diff
@@ -569,7 +570,8 @@ func TestMemMaxDepth(t *testing.T) {
 			}
 
 			_, _, err = vm.MemUsage(
-				NewMemUsageContext(vm, tc.expectedDepth+1, memUsageLimit, arrLenThreshold, objPropsLenThreshold, TestNativeMemUsageChecker{}),
+				// todo need to add 2 since Object is lazy loaded it will add onto the expected depth
```

Comment:
Might need to just make this a formal comment, since Object is lazy loaded there is 1 extra depth